Geometry/MuonSimData/data/muonProdCuts/2021/v3/muonProdCuts.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<DDDefinition>
+ <SpecParSection label="ProductionCuts" eval="true">
+  <SpecPar name="Muon">
+   <PartSelector path="//MUON"/>
+   <Parameter name="CMSCutsRegion" value="Muon" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*cm"/>
+   <Parameter name="ProdCutsForElectrons" value="1.0*cm"/>
+   <Parameter name="ProdCutsForPositrons" value="1.0*cm"/>
+  </SpecPar>
+  <SpecPar name="MuonIron">
+   <PartSelector path="//MGNT"/>
+   <PartSelector path="//YBSepar.*"/>
+   <PartSelector path="//COI.*"/>
+   <PartSelector path="//COCF"/>
+   <PartSelector path="//YB.*"/>
+   <PartSelector path="//ZStopSpace"/>
+   <PartSelector path="//SupportTubeForHE.*"/>
+   <PartSelector path="//YN.*"/>
+   <PartSelector path="//YE1.*"/>
+   <PartSelector path="//YE2.*"/>
+   <PartSelector path="//YE3.*"/>
+   <!-- if you want to re-stage YE4, then remove the comments around this element   <PartSelector path="//YE4.*"/>   -->
+   <PartSelector path="//ShieldingME4.*"/>
+   <PartSelector path="//ShieldingBoronPoly.*"/>
+   <PartSelector path="//ME11SupportDisk"/>
+   <Parameter name="CMSCutsRegion" value="MuonIron" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*cm"/>
+   <Parameter name="ProdCutsForElectrons" value="1.0*mm"/>
+   <Parameter name="ProdCutsForPositrons" value="1.0*mm"/>
+  </SpecPar>
+  <SpecPar name="MuonChamber">
+   <PartSelector path="//MB.*N"/>
+   <PartSelector path="//MB.*P"/>
+   <PartSelector path="//ME.*Space"/>
+   <PartSelector path="//RR.*"/>
+   <PartSelector path="//GE.*"/>
+   <Parameter name="CMSCutsRegion" value="MuonChamber" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*mm"/>
+   <Parameter name="ProdCutsForElectrons" value="0.002*mm"/>
+   <Parameter name="ProdCutsForPositrons" value="0.002*mm"/>
+  </SpecPar>
+  <SpecPar name="MuonSensitive_DT-CSC">
+   <!-- 1 keV for gamma/e-, 1 MeV for e+ -->
+   <PartSelector path="//MB.*PhiGas"/>
+   <PartSelector path="//MB.*ZGas"/>
+   <PartSelector path="//ME.*ActiveGasVol"/>
+   <PartSelector path="//GHA.*"/>
```

Comment:
This is the GEM sensitive volume right? I think we want to remove this, and keep consistent with the other geometries

Reply:
GHA* corresponds to the sensitive part and this is consistent with other product file - mainly the ones corrected fro Phase2

Reply:
Yes, but this is for CSC, and as Tim said today, the CSC digitizer is internally doing a microsimulation from the simhits, whereas in GEM we have a simple efficiency based digitizer that has been adapted to the existing simulation, which is using the MuonChamber cuts. So, I think it would be better to be consistent with the previous xmls. We are also planning to update the digitizer later on, maybe @jshlee could comment more on that.